lib/util.js

```diff
@@ -0,0 +1,70 @@
+const fs = require('fs');
+const exec = require('child_process').exec;
+const Docker = require('dockerode');
+const docker = new Docker();
+
+const MINUTES_IN_HOUR = 60;
+
+function nameMeetsCriteria(containerName, criteriaName) {
+    if (criteriaName === '*') {
+        return true
+    } else {
+        return containerName.includes(criteriaName)
+    }
+}
+
+function ageMeetsCriteria(containerAge, criteriaAge) {
+    return containerAge >= criteriaAge
+}
+
+function containerMeetsCriteria(container, criteria) {
+    containerName = container.Names[0];
+    containerAge = parseInt(container.Status.replace(/^\D+/g, ''));
+    criteriaName = criteria.name;
+    criteriaAge = criteria.age;
+
+    if (nameMeetsCriteria(containerName, criteriaName) && ageMeetsCriteria(containerAge, criteriaAge)) {
+        return true
+    } else {
+        return false
+    }
+}
+
+function updateCriteria(config) {
+    fs.writeFile('config/default.json', JSON.stringify(config, null, 2), function writeJSON(error) {
+        if (error) return console.log(error);
+    });
+}
+
+async function killContainers(criteria) {
+    docker.listContainers((error, containers) => {
+        if (containers) {
+            containers.forEach(async container => {
+                if (containerMeetsCriteria(container, criteria)) {
+                    await docker.getContainer(container.Id).remove({ force: true });
+                    console.log(`Container: ${container.Id} deleted successfully.`);
+                }
+            })
+        }
+    })
+}
+
+async function restartApp() {
+    await exec('pm2 start lib/server.js || pm2 restart all', 
+        (error, stdout, stderr) => {
+            if (error) 
+                console.log(error);
+            else
+                console.log('Container sweep started!');
+        }
+    );
```

Comment:
This should work for now, I suggest setting a name for it when running with pm2 (`--name container-sweep`) so you will be able to just restart this process in pm2 and not everything else.

another related thing is this is assuming pm2 is installed on the system. It might be good to install pm2 as a dependency and run `../node_modules/.bin/pm2` instead

Reply:
I think next-level improvement might be change this so instead of restarting the process completely it would use [`fs.promises.watch`](https://nodejs.org/docs/latest/api/fs.html#fspromiseswatchfilename-options) to monitor if config has changed or not and simply read new config when it finds a change. (current approach works for now, so this is not a high priority improvement if it takes a long time to implement)

Reply:
I will look into this but I have made the change to name the process using the `--name` option in pm2. I've also installed it as a project dependency so when a user clones the repo they can just run `npm i`.